genet/schedule_elements.py

```diff
@@ -3769,12 +3890,14 @@ def write_to_csv(self, output_dir, gtfs_day="19700101", file_extention="csv"):
             df.to_csv(file_path)
         self.write_extras(output_dir)
 
-    def write_to_gtfs(self, output_dir, gtfs_day="19700101"):
-        """
-        Writes 'stops', 'routes', 'trips', 'stop_times', 'calendar' tables to CSV files
-        :param output_dir: folder to output txt files
-        :param gtfs_day: day used for GTFS when creating the network in YYYYMMDD format defaults to 19700101
-        :return: None
+    def write_to_gtfs(self, output_dir: str, gtfs_day: str = "19700101"):
+        """Writes 'stops', 'routes', 'trips', 'stop_times', 'calendar' tables to CSV files.
+
+        FIXME: Is this mixed up with `write_to_csv`?
```

Comment:
`write_to_csv` uses the same schema as when writing to GTFS (because there really isn't a better way to save transit in tabular format), the only difference is with GTFS you expect the tables to have `txt` extension.